src/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# helper library for Sphere Lebedev Rules
+add_subdirectory(sphere_lebedev_rule)
+
+# shared library for Maier Saupe Liquid Crystal simulations
+file(GLOB cpp_src CONFIGURE_DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/[a-zA-Z]*.cpp)
+
+add_library(MaierSaupe SHARED ${cpp_src})
+target_include_directories(MaierSaupe PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
I moved the `include` here because the Pitchfork convention defines the `include` directory as headers for _external_ use i.e. headers that will be installed and used by a different user. Your headers are never installed or used outside of this repo, so they should be in the `src` directory with the rest of the library. Obviously, if you don't like this, you can move them back and then change the include_directories you attach to your library target.